File: BuildATree/src/buildatree/Jugador.java
package buildatree;

public class Jugador {
    /***
     * Envia recoge la información de los tokens recibidos por el jugador
     */

    int valor;
    String tipo;
    String nombre;

    public int getValor() {
        return valor;
    }

    public void setValor(int puntaje) {
        this.valor = puntaje;
    }

    public String getTipo() {
        return tipo;
    }

    public void setTipo(String tipo) {
        this.tipo = tipo;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }
}

File: Scripts/Stest/Jugador.cs
using System;
using UnityEngine;

[Serializable] 
public class Jugador
{
    /// <summary>
    /// Sends the information of the tokens to the server
    /// </summary>
    public int valor;
    public string tipo;
    public string nombre;

    public Jugador(int valor,string tipo, string nombre)
    {
        this.tipo = tipo;
        this.nombre = nombre;
        this.valor = valor;
    }

    public int getValor()
    {
        return valor;
    }

    public void setValor(int valor)
    {
        this.valor = valor;
    }

    public string getTipo()
    {
        return tipo;
    }

    public void setTipo(string tipo)
    {
        this.tipo = tipo;
    }
    public string getNombre()
    {
        return nombre;
    }

    public void setNombre(string nombre)
    {
        this.nombre = nombre;
    }

}
